rotkehlchen/inquirer.py

```diff
@@ -421,6 +439,7 @@ def find_price(
             to_asset: Asset,
             ignore_cache: bool = False,
             skip_onchain: bool = False,
+            is_root: bool = True,
```

Comment:
explaing what `is_root` argument is in the docstring. And perhaps `is_root` is not the right name here since you seem to only set it to `False` if called from `query_current_price`. So perhaps specify `coming_from_current_price` as the name of the boolean?